s3-proxy/nginx.conf

```diff
@@ -69,6 +69,11 @@ http {
             add_header 'Content-Type' 'text/plain';
             return 200 'S3 Proxy';
         }
+
+        location = /robots.txt {
+            add_header 'Content-Type' 'text/plain';
+            return 200 'User-agent: *\nDisallow: /\n';
```

Comment:
I was wondering if we need a terminating `\n`. Most examples I saw don't have it but I doubt it matters.

Reply:
Pretty sure it doesn't matter - but in general, files should end with a `\n`